src/ACF/Block_Config.php

```diff
@@ -4,41 +4,39 @@
 namespace Tribe\Libs\ACF;
 
 abstract class Block_Config {
-	public const NAME        = '';
-	public const CONTENT_TAB = 'content';
-	public const SETTING_TAB = 'settings';
+	public const NAME = '';
 
 	/**
-	 * @var Field_Collection
+	 * @var array
```

Comment:
Why did this change to `array`?

Reply:
I've cleaned this up in the latest but ultimately an array is easier to work with and is consistent with other classes. Specifically in the `With_Sub_Fields` trait, I wanted to make that use `Field_Collection` but that means every class that uses it has to create an instance somewhere. Array was easier here.